README.md

```diff
@@ -152,6 +173,7 @@ Execute `npm run test` and update the [tests](https://github.com/etienne-martin/
 ## Authors
 
 * **Etienne Martin** - *Initial work* - [etiennemartin.ca](http://etiennemartin.ca/)
+* **Philipp Katz** - *Connect to browser endpoint* - [gitlab.com/qqilihq](https://gitlab.com/qqilihq)
```

Comment:
Don't forget to add a `contributors` section with your info right below the `author` property in the `package.json`.

```
{
  "author": {
    "name": "Etienne Martin",
    "url": "http://etiennemartin.ca/"
  },
  "contributors": [
    "Philipp Katz (https://gitlab.com/qqilihq)"
  ]
}
```